source/extended-json/extended-json-query.md

```diff
@@ -0,0 +1,56 @@
+# Extended JSON Query
+
+## Abstract
```

Comment:
I think the abstract should be rewritten. Here is my attempt to write an abstract as I understand it:

This document describes a standard format which can be used for representing BSON documents textually. The format is designed with two principles in mind:
- The ability to represent any legal BSON document in a string format uniquely. That is, any legal BSON document should have exactly one representation in the BSON Text Format. Conversely, a valid BSON Text Format string must be uniquely convertible to a particular byte string which is a valid BSON document.
- Human readability. A primary use case of the format is for describing MQL correctness tests declaratively, and we expect those tests to often by inspected by server developers.

I actually think bullet number 1 is pretty interesting. I am confident that we want a given BSON Text Format string to always uniquely represent a BSON document (where uniqueness is defined in terms of binary equality). I'm less sure of whether we require a single unique BSON Text Format string for a BSON document -- trailing commas for the last element being one example of where we could choose to relax this.